keps/0001-kubernetes-enhancement-proposal-process.md

```diff
@@ -145,7 +144,7 @@ These KEPs will be owned by SIG-architecture and should be seen as a way to comm
 ### KEP Template
 
 <b>
-The template for a KEP is precisely defined [here](/keps/NNNN-kep-template).
+The template for a KEP is precisely defined [here](https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template).
```

Comment:
The original formatting is preferred here.  This will always link to the master repo instead of a fork or copy you're working with. External GitHub links or links using our shortener (git.k8s.io) are used when linking to items outside of the current repo 👍 
ref: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#links